src/components/vive-controls.js

```diff
@@ -105,6 +106,7 @@ module.exports.Component = registerComponent('vive-controls', {
     el.removeEventListener('touchstart', this.onButtonTouchStart);
     el.removeEventListener('model-loaded', this.onModelLoaded);
     el.removeEventListener('axismove', this.onAxisMoved);
+    this.controlllerEventsActive = false;
```

Comment:
I think that removing / adding event listeners in the controller specific controls is not strictly necessary if `tracked-controls` pauses / plays as expected.